__tests__/testHelpers.js

```diff
@@ -20,3 +25,34 @@ export const provideState = (node, initialState = {}) => {
   const store = createMockStore(initialState);
   return provideStore(node, store);
 };
+
+const spunkyInitialState = {
+  batch: false,
+  progress: progressValues.INITIAL,
+  rollbackProgress: null,
+  loadedCount: 0,
+  data: null,
+  error: null
+};
+
+export const spunkyKey = 'spunky';
+
+export const mockSpunkyLoaded = (data, { loadedCount = 1 } = {}) => ({
+  ...spunkyInitialState,
+  progress: progressValues.LOADED,
+  loadedCount,
+  data
+});
+
+export const mockSpunkyLoading = ({ loadedCount = 0 } = {}) => ({
+  ...spunkyInitialState,
+  progress: progressValues.LOADING,
+  loadedCount
+});
+
+export const mockSpunkyFailed = (error, { loadedCount = 0 } = {}) => ({
```

Comment:
My plan is to eventually move these into the spunky package, but I want to make sure they are sufficient for our use cases going forward first.